hack/loadtest/test-crp.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ClusterResourcePlacement
+metadata:
+  name: crp
+spec:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 15%
+      maxSurge: 35%
+      unavailablePeriodSeconds: 45
+    revisionHistoryLimit: 15
```

Comment:
Hi Britania! For linting purposes, would you mind adding a blank line at the end of this file?

Reply:
Same for the rest few YAML files as well.